sdks/wasm/DebuggerTestSuite/ExceptionTests.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+using Xunit;
+using WebAssembly.Net.Debugging;
+
+namespace DebuggerTests
+{
+
+    public class ExceptionTests : DebuggerTestBase {
+
+        [Fact]
+        public async Task ExceptionTestAll () {
+            var insp = new Inspector ();
+			//Collect events
+			var scripts = SubscribeToScripts(insp);
+			int line = 9;
+			int col = 5;
+			string entry_method_name = "[debugger-test] DebuggerTests.ExceptionTestsClass:TestExceptions";
+
+			await Ready();
+			await insp.Ready (async (cli, token) => {
+				ctx = new DebugTestContext (cli, insp, token, scripts);
+				var debugger_test_loc = "dotnet://debugger-test.dll/debugger-exception-test.cs";
+
+                await SetPauseOnException("all");
+
+                var eval_expr = "window.setTimeout(function() { invoke_static_method ("
+							+ $"'{entry_method_name}'"
+						+ "); }, 1);";
+
+				var pause_location = await EvaluateAndCheck (eval_expr, debugger_test_loc, line, col, "run");
```

Comment:
What's the expected behavior for this? Does the debugger pause even if we were not stepping?

Also, you should inspect that exception object, and check where this paused.

Reply:
2. Does "all" mean that the debugger will pause whenever an exception is thrown? Caught, and uncaught? if so, then we need to test that case too.

Reply:
Yes, both of them. Okay, I'll create another test.

Reply:
I don't think I need to check the exception object, because I'm already checking the line number of callstack, why should I check the exception object?

Reply:
Just to ensure that we can get the exception object, so that it can be shown by the debugger, and can be inspected by the user.

Reply:
Okay :)

Reply:
Done!